mne_qt_browser/_pg_figure.py

```diff
@@ -3138,7 +3138,10 @@ def _update_data(self):
 
     def _get_zscore(self, data):
         # Reshape data to reasonable size for display
-        max_pixel_width = QApplication.desktop().screenGeometry().width()
+        if QApplication.desktop() is None:
+            max_pixel_width = 1920  # defaul=FHD
```

Comment:
I know people who have had 4K for years, and I think it's more the standard nowadays...

Reply:
Ok, I wish I had one too... ;)